substrate/frame/collective/src/lib.rs

```diff
@@ -1119,7 +1122,7 @@ impl<T: Config<I>, I: 'static> InitializeMembers<T::AccountId> for Pallet<T, I>
 			assert!(<Members<T, I>>::get().is_empty(), "Members are already initialized!");
 			let mut members = members.to_vec();
 			members.sort();
-			<Members<T, I>>::put(members);
+			<Members<T, I>>::put(BoundedVec::try_from(members).expect("Too many members!"));
```

Comment:
Handling too many members by panicing? 🙈

Reply:
I did not put the assert there, and seeing that, I thought it was a method that should panic when something goes wrong. 
Should I fix both lines?

Reply:
Let's use `defensive` here too

Reply:
Ah yep that does seem like a pattern, seems it's OK to panic but only at genesis. My bad.

Let's add a doc comment to this method though making it clear that it can panic.